config.py

```diff
@@ -1,2 +1,4 @@
-ELASTIC_SEARCH_URL = 'localhost:9200'
-TEST_ELASTIC_SEARCH_URL = 'localhost:9200'
+import os
+
+ELASTIC_SEARCH_URL = os.environ.get('ELASTIC_SEARCH_URL', 'http://localhost:9200')
+TEST_ELASTIC_SEARCH_URL = os.environ.get('TEST_ELASTIC_SEARCH_URL', 'http://localhost:9200')
```

Comment:
I don't think you need to fix this now because it looks like you never use a real Elasticsearch index in the tests, but having the production and test URLs be the same is a problem. I filed https://jira.nypl.org/browse/SIMPLY-2612 to cover this.